pages/map.js

```diff
@@ -122,8 +122,8 @@ export default function Map({ data }) {
             <Button
               onClick={resetFilter}
               text="Clear/Reset Filters"
-              primary={false}
               disable={selectedTags.size == 0 ? true : false}
+              primary={false}
```

Comment:
Not sure if this move is required?

Reply:
Yes, it was not required, it happened when I pulled changes from main branch.

Reply:
Probably best to revert these changes, to reduce conflicts for others